--- a/drivers/input/touchscreen/Kconfig	2020-05-31 01:31:58.956762724 +0530
+++ b/drivers/input/touchscreen/Kconfig	2020-05-31 02:57:21.636821688 +0530
@@ -15,6 +15,14 @@
 	def_tristate INPUT
 	depends on INPUT
 
+config TOUCHSCREEN_FT5X06
+	tristate "Focaltech FT5X06 5 point touchscreen"
+	select I2C
+	default y
+	help
+	  If you say yes here you get touchscreen support through
+	  FocalTech's FT5X06 controller.
+
 config TOUCHSCREEN_88PM860X
 	tristate "Marvell 88PM860x touchscreen"
 	depends on MFD_88PM860X
