--- a/arch/arm/boot/dts/r8a7743-skrzg1m.dts	2020-05-30 21:04:00.828577659 +0530
+++ b/arch/arm/boot/dts/r8a7743-skrzg1m.dts	2020-05-30 21:42:50.136604470 +0530
@@ -225,6 +225,11 @@
 		renesas,function = "i2c2";
 	};
 
+	i2c4_pins: i2c4 {
+		renesas,groups = "i2c4_c";
+		renesas,function = "i2c4";
+	};
+
 	usb0_pins: usb0 {
 		renesas,groups = "usb0";
 		renesas,function = "usb0";
@@ -397,6 +402,22 @@
 	};
 };
 
+&i2c4 {
+	pinctrl-0 = <&i2c4_pins>;
+	pinctrl-names = "default";
+
+	status = "okay";
+	clock-frequency = <400000>;
+
+    touch: edt-ft5506@38 {
+        compatible = "ft5x06,ft5x06-touch";
+        reg = <0x38>;
+        interrupt-parent = <&irqc0>;
+        interrupts = <1 IRQ_TYPE_LEVEL_LOW>;
+    };
+
+};
+
 &i2c6 {
 	status = "okay";
 	clock-frequency = <100000>;
